src/components/Navbar.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Link } from 'gatsby'
-import logo from '../../static/img/smallLogo.png'
+import logo from '../img/header-icon.png'
```

Comment:
why not the full icon? I liked that one for the navbar better.

Reply:
I'm just following the spec 🤷‍♂️

Reply:
reported